tests/unit/test_hnswlib.cpp

```diff
@@ -98,44 +100,60 @@ TEST_F(HNSWLibTest, resizeNAlignIndex_initailCapacity_is_big) {
 
     float a[dim];
 
-    // add up to blocksize + 1
+    // add up to blocksize + 1 = 3 + 1 = 4
     for (size_t i = 0; i < bs + 1; i++) {
         for (size_t j = 0; j < dim; j++) {
             a[j] = (float)i;
         }
         VecSimIndex_AddVector(index, (const void *)a, i);
     }
 
-    // the capcity shouldnt be changed
+    // The capcity shouldnt be changed. (32)
     ASSERT_EQ(reinterpret_cast<HNSWIndex *>(index)->getHNSWIndex()->getIndexCapacity(), n);
 
-    // delete random vector, to get size % block_size == 0
+    // Delete random vector, to get size % block_size == 0. size = 3
     VecSimIndex_DeleteVector(index, 1);
 
-    ASSERT_EQ(reinterpret_cast<HNSWIndex *>(index)->getHNSWIndex()->getIndexSize(), bs);
+    // Index size = bs = 3.
```

Comment:
Why did you remove the test itself? 
I would add here:
`ASSERT_EQ(VecSimIndex_IndexSize(index), bs)`
instead of the comment

Reply:
its in line 118?

Reply:
right! sorry :)